packages/strapi-versioning-plugin/src/extensions/content-manager/controllers/Versions.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+
+module.exports = {
```

Comment:
You could write Controllers in typescript

Reply:
done